### ISSUE TITLE
NTR: Pain associated with bowel movements (GI, feeding, vomiting & dehydration)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7888]
New term request Pain associated with bowel movements (segal_211022153159): Defecation  UMLS: 4540981

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000222
- label: Painful defecation
- parents: [Pain (HP:0012531)](https://hpo.jax.org/app/browse/term/HP:0012531)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort that occurs when passing stool.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7888
- synonyms: Pain associated with bowel movements(NONE)
